kubernetes/django_config.yaml.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Is there a reason not to put all of these in the `Secret` instead (to reduce the number of objects we have to manage)? I think at least the `EMAIL_HOST_USER` should go there since it's secret.